### ISSUE TITLE
Suggest PrivateAssets option set to all by default

### ISSUE THREAD

**Issue by NiccoMlt** [kaby76/Antlr4BuildTasks#11]
Hi, yesterday I exported an executable which uses under the hood ANTLR and generates parser, lexer and visitor code with the targets provided by this package and I noticed that also Antlr4BuildTasks.dll is exported in the publish folder.

Does this package also export code that should be bundled with the application?
If not, I think it would be better to suggest in the documentation to set [`PrivateAssets` option](https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#controlling-dependency-assets) to `all` in the package reference:

```xml
<ItemGroup>
  <PackageReference Include="Antlr4.Runtime.Standard" Version="4.9.0" />
  <PackageReference Include="Antlr4BuildTasks" Version="8.9">
    <PrivateAssets>all</PrivateAssets>
  </PackageReference>
</ItemGroup>
```

This applies to [README](https://github.com/kaby76/Antlr4BuildTasks#conversion-from-antlr4codegeneratorantlr4runtime-to-antlr4buildtasksantlr4runtimestandard) but also to [Antlr4BuildTasks.Templates](https://github.com/kaby76/Antlr4BuildTasks/tree/master/Templates) and to [kaby76/testantlr](https://github.com/kaby76/testantlr).

Also, I don't know if it is possible to configure the NuGet package to install itself with this default when installed via GUI, in a similar way analyzers do.

**Comment by kaby76** [comment]
Yes, you are correct! Thanks for the info. I'll add it to the doc and in the templates and examples.

**Comment by kaby76** [comment]
Fixed in templates-v8.9.1 and updated readme.md.
